sql/moz-fx-data-shared-prod/search_terms_derived/suggest_impression_aggregated/search_queries_daily_v1/init.sql

```diff
@@ -0,0 +1,15 @@
+CREATE TABLE IF NOT EXISTS
+  `moz-fx-data-shared-prod.search_terms_derived.search_queries_daily_v1`
```

Comment:
@rebecca-burwei and I spoke about naming just now and agreed on `aggregated_search_terms_daily` here.